central/alert/datastore/internal/search/searcher_impl.go

```diff
@@ -130,7 +132,13 @@ func convertAlert(alert *storage.ListAlert, result search.Result) *v1.SearchResu
 ///////////////////////////////////////////////
 
 func formatSearcher(unsafeSearcher blevesearch.UnsafeSearcher) search.Searcher {
```

Comment:
fork this out. PaginatedSearcher is entirely redundant in postgres, and defaultsortedsearcher should not be required. It should come from clients (service layer or other consumers)